NitroxServer/Serialization/World/WorldManager.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using NitroxModel.Helper;
+
+namespace NitroxServer.Serialization.World;
+
+public static class WorldManager
+{
+    public static readonly string SavesFolderDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "Nitrox", "saves");
+
+    public static readonly string[] WorldFiles = { "BaseData.json", "EntityData.json", "PlayerData.json", "Version.json", "WorldData.json" };
```

Comment:
This is way too dependent on how Nitrox works from the outside perspective (4 files named x, y, z, ... with .json), if one day someone adds or change something related to data serialization, this is likely to become broken

Reply:
I recently added improvements. Do they solve this issue?